ios/CodePush/CodePushPackage.m

```diff
@@ -231,18 +245,38 @@ + (void)downloadPackage:(NSDictionary *)updatePackage
                                                             }
                                                         }
                                                         
-                                                        if (isDiffUpdate && ![CodePushUpdateUtils verifyHashForDiffUpdate:newUpdateFolderPath
-                                                                                                             expectedHash:newUpdateHash
-                                                                                                                    error:&error]) {
-                                                            if (error) {
-                                                                failCallback(error);
+                                                        if(isDiffUpdate){
+                                                            CPLog(@"Applying diff update.");
+                                                        }else{
```

Comment:
Just style issue to use spaces between `}else{` or probably use [Ternary ](https://en.wikipedia.org/wiki/Ternary_operation)operator here

Reply:
@max-mironov got it!